### PR TITLE
Login UI

### PR DESCRIPTION
**Pull request by Khukhuna**
Part of GCI :heart: :balloon: 

As task required, added login ui from given wireframe

If any change is required, please let me know :smiley:.

### CODE REVIEW

**Comment by sidhyatikku** [comment]
Wow this looks very nice. We can be a good team in the future 🥂

**Comment by the-dagger** [comment]
Damn, real nice!
@nikitadhiman Is this good to go?

**Comment by Khukhuna** [comment]
I'm still working on Responsiveness &  connection Django server to implement login function

**Comment by nikitadhiman** [comment]
And yes, we need it to be responsive. Let me know when you are done with responsive part

**Comment by prateekiiest** [comment]
Great work @Khukhuna.
I approve of this.

File: package.json [hunk 1]
```diff
@@ -22,8 +22,9 @@
     "@angular/router": "^6.1.0",
     "ag-grid-angular": "^19.0.0",
     "ag-grid-community": "^19.0.0",
-    "bootstrap": "^4.1.3",
+    "bootstrap": "^3.3.7",
```
Comment (nikitadhiman) [inline package.json]: why lower version?

Comment (Khukhuna) [reply package.json]: Done :smile: 
Moved v3.3.7 to v4.1.3

**Comment by nikitadhiman** [comment]
Yes, sure.

**Comment by sidhyatikku** [comment]
Okay cool, come on DMs on gitter @nikitadhiman

**Comment by nikitadhiman** [comment]
Sure

**Comment by RaghavAwasthi** [comment]
@Khukhuna  This is damn amazing.

**Pull request merged**